### PR TITLE
Complete and improve transition tokens

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**

### CODE REVIEW

File: tokens/properties/alias.json [hunk 1]
```diff
@@ -422,7 +422,8 @@
                     "value": "{box-shadow.inset.style.thin.value} {color.utility.red.50.value}"
                 },
                 "button-primary": {
-                    "value": "{box-shadow.inset.style.thick.value} {color.base.100.value}"
+                    "value": "{box-shadow.inset.style.thick.value} {color.base.100.value}",
+                    "comment": "Only applies to filled primary buttons"
```
Comment (tzhelyazkova) [inline tokens/properties/alias.json:426]: Is this comment meant to be shown in Storybook?

File: tokens/properties/global.json [hunk 2]
```diff
@@ -306,6 +306,14 @@
...
             "box-shadow": {
                 "value": "box-shadow"
+            },
+            "color": {
+                "value": "color"
+            }
+        },
+        "timing function": {
+            "ease": {
+                "value": "ease"
```
Comment (tzhelyazkova) [inline tokens/properties/global.json:316]: This is currently not showing in Storybook. We need a new mdx file (09-transition-04...).

Comment (SaiSan-WMDE) [reply tokens/properties/global.json:316]: agh! true

File: tokens/properties/alias.json (hunk 1 above)
Comment (SaiSan-WMDE) [reply tokens/properties/alias.json:426]: Nope!

**Pull request merged**